lagom10-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/lagom/Env.scala

```diff
@@ -0,0 +1,39 @@
+package com.typesafe.conductr.bundlelib.lagom
+
+import com.typesafe.config.{ Config, ConfigFactory }
+
+import scala.collection.JavaConverters._
+
+/**
+ * Provides functions to set up the Play environment in accordance with what ConductR provides.
+ */
+object Env extends com.typesafe.conductr.bundlelib.scala.Env {
+
+  /**
+   * Provides various Lagom related properties.
+   */
+  def asConfig: Config =
+    ConfigFactory.parseMap(lagomCassandraKeyspace.asJava)
+
+  private final val CassandraKeyspaceNameRegex = """^("[a-zA-Z]{1}[\w]{0,47}"|[a-zA-Z]{1}[\w]{0,47})$"""
+  private def isValidKeyspaceName(name: String): Boolean =
+    name.matches(CassandraKeyspaceNameRegex)
+
+  private def lagomCassandraKeyspace: Map[String, String] =
+    sys.env.get("BUNDLE_NAME").fold(Map.empty[String, String]) { bundleName =>
+      val keyspace =
+        if (isValidKeyspaceName(bundleName))
+          bundleName
+        else {
+          // I'm confident the normalized name will work in most situations. If it doesn't, then
+          // the application will fail at runtime and users will have to provide a valid keyspace
+          // name in the application.conf
```

Comment:
Indeed, but in dev mode users can currently change the keyspace in the build, as you can see [here](https://github.com/typesafehub/lagom/blob/master/docs/manual/devmode/detailedTopics/code/build-cassandra-opts2.sbt#L5). Regardless, even if the keyspace is provided in the `application.conf`, a user may use a different config file in production, so it may still experience a failure. If that's ok, then I think it's fine to go ahead with what you have here.